### DISCUSSION TITLE
小狼毫0.16的更新，如果遇到问题，可以跟我一样

### DISCUSSION THREAD

**Discussion by jqrxjkpyjpn** [rime/home#1586] [General]
本来还想着，万年不更新，这次更新了什么好功能，结果更新完之后，电脑直接灰屏。
重启N次都一样，什么事都干不了。
alt+tab也切换不了窗口，任务管理器调出来了也没用，什么都干不了，就只有个能看到鼠标的灰屏。
另：如果碰到和我一样的朋友，可以在进桌面的第一时刻，右下角图标里，把小狼毫退出服务，然后用别的输入法，我暂时就是这样解决的。
（win10专业版，1909。18363。1556）

**Comment by LEOYoon-Tsaw** [comment]
說說你具體的配置吧，用了lua嗎？有log嗎？
單純抱怨也無助於解決問題是吧

**Reply by jqrxjkpyjpn** [reply]
我是个普通用户，不是程序员，用它打五笔的，不懂lua是什么，也不知道自己用没用。
这些是我找到的日志文件。
[rime.weasel.Y.YD.log.INFO.20240608-082554.3680.log](https://github.com/user-attachments/files/15746041/rime.weasel.Y.YD.log.INFO.20240608-082554.3680.log)
[rime.weasel.Y.YD.log.WARNING.20240608-082554.3680.log](https://github.com/user-attachments/files/15746042/rime.weasel.Y.YD.log.WARNING.20240608-082554.3680.log)
[rime.weasel.Y.YD.log.INFO.20240608-082556.12340.log](https://github.com/user-attachments/files/15746043/rime.weasel.Y.YD.log.INFO.20240608-082556.12340.log)
[rime.weasel.Y.YD.log.INFO.20240608-082659.10584.log](https://github.com/user-attachments/files/15746044/rime.weasel.Y.YD.log.INFO.20240608-082659.10584.log)
[rime.weasel.Y.YD.log.INFO.20240608-082922.10884.log](https://github.com/user-attachments/files/15746045/rime.weasel.Y.YD.log.INFO.20240608-082922.10884.log)
[rime.weasel.Y.YD.log.INFO.20240608-083117.11604.log](https://github.com/user-attachments/files/15746046/rime.weasel.Y.YD.log.INFO.20240608-083117.11604.log)
[rime.weasel.Y.YD.log.INFO.20240608-083151.14480.log](https://github.com/user-attachments/files/15746047/rime.weasel.Y.YD.log.INFO.20240608-083151.14480.log)
[rime.weasel.Y.YD.log.INFO.20240608-084434.12088.log](https://github.com/user-attachments/files/15746048/rime.weasel.Y.YD.log.INFO.20240608-084434.12088.log)
[rime.weasel.Y.YD.log.INFO.20240608-084459.16296.log](https://github.com/user-attachments/files/15746049/rime.weasel.Y.YD.log.INFO.20240608-084459.16296.log)
[rime.weasel.Y.YD.log.WARNING.20240608-084500.16296.log](https://github.com/user-attachments/files/15746050/rime.weasel.Y.YD.log.WARNING.20240608-084500.16296.log)
[rime.weasel.Y.YD.log.INFO.20240608-084503.15840.log](https://github.com/user-attachments/files/15746051/rime.weasel.Y.YD.log.INFO.20240608-084503.15840.log)
[rime.weasel.Y.YD.log.INFO.20240608-084645.8352.log](https://github.com/user-attachments/files/15746052/rime.weasel.Y.YD.log.INFO.20240608-084645.8352.log)
[rime.weasel.Y.YD.log.INFO.20240608-084830.7368.log](https://github.com/user-attachments/files/15746053/rime.weasel.Y.YD.log.INFO.20240608-084830.7368.log)
[rime.weasel.Y.YD.log.INFO.20240608-084859.14348.log](https://github.com/user-attachments/files/15746054/rime.weasel.Y.YD.log.INFO.20240608-084859.14348.log)
[rime.weasel.Y.YD.log.INFO.20240608-085033.10528.log](https://github.com/user-attachments/files/15746055/rime.weasel.Y.YD.log.INFO.20240608-085033.10528.log)
[rime.weasel.Y.YD.log.INFO.20240608-085107.14580.log](https://github.com/user-attachments/files/15746056/rime.weasel.Y.YD.log.INFO.20240608-085107.14580.log)
[rime.weasel.Y.YD.log.INFO.20240608-091447.8092.log](https://github.com/user-attachments/files/15746057/rime.weasel.Y.YD.log.INFO.20240608-091447.8092.log)
[rime.weasel.Y.YD.log.INFO.20240608-091642.13296.log](https://github.com/user-attachments/files/15746058/rime.weasel.Y.YD.log.INFO.20240608-091642.13296.log)

**Reply by LEOYoon-Tsaw** [reply]
行吧，大家都是用愛發電，你能做的貢獻就是詳細的描述你遇到的情況，幫助別人排查問題。

**Comment by baoang** [comment]
更新日志上似乎说明了，如果从之前的版本升级，有一些注意事项，其中包括要选退出程序等等。
